en/docs/get-started/configure-users-and-roles.md

```diff
@@ -66,4 +66,79 @@ Now you have started the servers, let’s create the users and define their perm
     ![set_new_user_roles](../assets/img/get-started/quick-start-guide/set-new-user-roles.png)
         
     iii. Click **Finish**.
+
+## Configuring SMS OTP Authenticator
+
+1. Sign in to the [Management Console](https://localhost:9446/carbon) as an administrator
+
+2. Adding SMS OTP Identity Provider:
+
+     - Navigate to the Main menu to access the Identity menu and select **Identity Providers** -> **Add**.
```

Comment:
Suggestion:
Go to the **Main menu > Identity > Identity Providers > Add**

Reply:
fix review comments